frb/data/Galaxies/190102/FRB190102_host.json

```diff
@@ -3,27 +3,28 @@
     "cosmo": "Planck15",
     "dec": -79.47566666666667,
     "derived": {
-        "AV_nebular": 0.09427130033655942,
-        "EBV_photom": 0.10874369730986228,
-        "EBV_photom_err": 0.046784733194462205,
-        "EBV_spec": 0.6094787256151387,
-        "Lnu_r": 3.997132381967197e+21,
-        "Lnu_r_err": 2.6143450055200063e+20,
-        "M_r": -19.90437132999839,
-        "M_r_err": 0.07101313524233827,
-        "Mstar": 3178839293.9744,
-        "Mstar_err": 2129235298.7722566,
-        "Mstar_spec": 5104978902.866098,
```

Comment:
Error larger than value. Is this due to a conversion from log to linear scale?

Reply:
There is no conversion.